crates/cairo-lang-dojo/src/query.rs

```diff
@@ -42,11 +44,27 @@ impl Query {
                     ast::PathSegment::Simple(segment) => {
                         let var_prefix = segment.as_syntax_node().get_text(db).to_ascii_lowercase();
 
-                        // TODO(https://github.com/dojoengine/dojo/issues/27): Properly compute component id
-                        let component_id = format!(
-                            "{:#x}",
-                            starknet_keccak(path.as_syntax_node().get_text(db).as_bytes())
+                        let class_hash = "0x00000000000000000000000000000000";
+                        let world_address = "0x00000000000000000000000000000000";
+
+                        // Component name to felt
+                        let component_name = path.as_syntax_node().get_text(db);
```

Comment:
I think this will give the struct name rather than the module. The module name feels more appropriate as the identifier (PositionComponent rather than Position) but might be tricky to get here (i think we need to handle the `use`)

We'll also want to handle fully qualified imports here, like `PositionComponent::Position`. Getting the text on the path will return the whole path I think

Reply:
Although, thinking about it more, maybe it is ok like this and we can enforce that component struct naming is globally unique in the world?

Reply:
Aye cool, we should still fix for absolute imports probably.